develop/dev-guide-build-cluster-in-cloud.md

```diff
@@ -154,7 +167,7 @@ mysql --connect-timeout 15 -u root -h <host> -P 4000 -p
   mysqlDataSource.setServerName("localhost");
   mysqlDataSource.setPortNumber(4000);
   mysqlDataSource.setDatabaseName("test");
-  mysqlDataSource.setUser("root");
+  mysqlDataSource.setUser("<prefix>.root");
```

Comment:
L152-L160 The `Local default cluster` tab is unnecessary.

Reply:
I've checked and confirmed with Icemap. He will make similar changes in the Chinese docs as well.